### PR TITLE
Clean up some things I noticed while doing another change

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description
This fixes some minor things I noticed while doing another change.

1) Uncomments an old test that wants to be run, but can't be, and marked it as "skipped", so that the code won't rot anymore.
2) Added the focus state to the short string version of a `FocusNode`
3) Added a missing piece of information to `findAncestorWidgetOfExactType`
## Related Issues
 - #4771

## Breaking Change

- [X] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/focus_manager.dart [hunk 1]
```diff
@@ -911,7 +911,10 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
 
   @override
   String toStringShort() {
-    return '${describeIdentity(this)}${debugLabel != null && debugLabel.isNotEmpty ? '($debugLabel)' : ''}';
+    final String extraData = '${debugLabel != null && debugLabel.isNotEmpty ? '$debugLabel ' : ''}'
+        '${hasFocus && !hasPrimaryFocus ? '[FOCUSED]' : ''}'
+        '${hasPrimaryFocus ? '[PRIMARY]' : ''}';
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/focus_manager.dart]: Wondering if it should still say FOCUSED in this case because it is still focused right?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/focus_manager.dart (hunk 1 above)
Comment (gspencergoog) [reply packages/flutter/lib/src/widgets/focus_manager.dart]: Yeah, but I thought that "[PRIMARY FOCUSED]' was too long for a "short" string. If you don't think so, I'm happy to add it in too.

File: packages/flutter/test/widgets/focus_manager_test.dart [hunk 2]
```diff
@@ -664,38 +664,38 @@ void main() {
...
           ' │ primaryFocusCreator: Container-[GlobalKey#00000] ← [root]\n'
           ' │\n'
-          ' └─rootScope: FocusScopeNode#00000(Root Focus Scope)\n'
+          ' └─rootScope: FocusScopeNode#00000(Root Focus Scope [FOCUSED])\n'
           '   │ IN FOCUS PATH\n'
-          '   │ focusedChildren: FocusScopeNode#00000\n'
+          '   │ focusedChildren: FocusScopeNode#00000([FOCUSED])\n'
           '   │\n'
-          '   ├─Child 1: FocusScopeNode#00000(Scope 1)\n'
+          '   ├─Child 1: FocusScopeNode#00000(Scope 1 )\n'
```
[the remark this answers is not in the document]
Comment (gspencergoog) [reply packages/flutter/test/widgets/focus_manager_test.dart]: Removed.

File: packages/flutter/lib/src/widgets/focus_manager.dart (hunk 1 above)
Comment (goderbauer) [reply packages/flutter/lib/src/widgets/focus_manager.dart]: I feel like being more explicit here would be ok. It's still pretty short :)

Comment (gspencergoog) [reply packages/flutter/lib/src/widgets/focus_manager.dart]: OK, done.

**Pull request merged**